def show_error_msg():
    print("에러가 발생 했습니다")

show_error_msg()

